### ISSUE TITLE
[jsx-curly-brace-presence] What do we do with HTML Entities like &nbsp;?

### ISSUE THREAD

**Issue by rafbgarcia** [yannickcr/eslint-plugin-react#2522]
Right now the code below fails, what should we do about it?

```jsx
<App>
  <Component />
  &nbsp;
</App>
```

Should we ignore HTML entities?

**Comment by ljharb** [comment]
If you have curly brace presence set to "always" for children, then I would expect it to require `{'&nbsp;'}`.

**Comment by rafbgarcia** [comment]
@ljharb but that would cause the output to be invalid because HTML entities produce special characters to the HTML document.
